configs/TBS_LUCID_FC/config.h

```diff
@@ -76,20 +79,20 @@
 #define UART3_TX_PIN                        PB11
 #define UART3_RX_PIN                        PB10
 
-#define UART4_TX_PIN                        NONE
-#define UART4_RX_PIN                        NONE
+#define UART4_TX_PIN                        PH3
+#define UART4_RX_PIN                        PH2
 
-#define UART5_TX_PIN                        PB6
-#define UART5_RX_PIN                        PB5
+#define UART5_TX_PIN                        PB9
+#define UART5_RX_PIN                        PD2
 
 #define UART7_TX_PIN                        PB4
 #define UART7_RX_PIN                        PB3
 
 #define UART8_TX_PIN                        PC2
 #define UART8_RX_PIN                        PC3
 
-#define I2C2_SCL_PIN                        PH2
-#define I2C2_SDA_PIN                        PH3
+#define I2C1_SCL_PIN                        PB6
+#define I2C1_SDA_PIN                        PC7
```

Comment:
Can we have PH0 / PH1 for I2C1 instead as to preserve using single GPIO port [for motors] - which also needs to be added to bus_i2c_atbsp_init.c?

Reply:
PH0 / PH1 are the external crystal pins, we probably wanna preserve those.

Reply:
![image](https://github.com/betaflight/config/assets/8344830/a915a353-674b-4617-b921-d4596e97b8f0)